grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -162,7 +163,7 @@ public GraknAdmin admin(){
 
     @Override
     public <T extends Concept> T buildConcept(Vertex vertex) {
-        return elementFactory.buildConcept(vertex);
+        return getElementFactory().buildConcept(vertex);
```

Comment:
Nah, this wasn't a request for it to change. Just wondering if there was a reason for it.